src/rust/ensogl/src/math/topology/unit.rs

```diff
@@ -261,6 +261,24 @@ impl AngleOps for i32 {
     }
 }
 
+impl AngleOps for Angle<Degrees> {
+    fn degrees(&self) -> Angle<Degrees> {
+        self.clone()
+    }
+
+    fn radians(&self) -> Angle<Radians> {
+        Angle::new(self.value.to_radians())
+    }
+}
+
+impl AngleOps for Angle<Radians> {
+    fn degrees(&self) -> Angle<Degrees> { Angle::new(self.value.to_degrees()) }
```

Comment:
Lets use multiline layout. We are using single-line layout only if we have many similar lines glued together and we want to show their similarity.